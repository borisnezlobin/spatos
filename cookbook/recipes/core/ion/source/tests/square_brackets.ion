echo "[${c::red,bold}asdf${c::default}]"
echo "[${c::red,bold}asdf${c::default}"
echo "${c::red,bold}asdf${c::default}]"