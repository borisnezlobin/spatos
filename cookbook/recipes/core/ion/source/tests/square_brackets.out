[[1;31masdf[39m]
[[1;31masdf[39m
[1;31masdf[39m]
